project/Dependencies.scala

```diff
@@ -66,4 +67,5 @@ object Dependencies {
 
   val scalaNativeTools = "org.scala-native" %% "tools" % scalaNativeVersion
   val scalaJsTools = "org.scala-js" %% "scalajs-tools" % scalaJsVersion
+  val mill = "com.lihaoyi" %% "mill-scalalib"	% millVersion
```

Comment:
probably should tag this as "provided"

Reply:
Yes, I think so. It depends on how mill implements plugin classloading, but if the mill classloader is the parent of your plugin classloader, it should be `Provided` :+1:

Reply:
Addressed in 4943ed5 (tested locally)